src/main/java/com/company/TwitterAppConfiguration.java

```diff
@@ -0,0 +1,60 @@
+package com.company;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dropwizard.Configuration;
+import org.hibernate.validator.constraints.NotEmpty;
+
+public class TwitterAppConfiguration extends Configuration {
+
+    @NotEmpty
```

Comment:
you should be using hibernate NotEmpty for a dropwizard configuration, hibernate doesn't come into play in this entire program/